pipeline/outputs/observe.md

```diff
@@ -0,0 +1,42 @@
+# Observe
+
+Observe employs the **http** output plugin, allowing you to flush your records [into Observe](https://docs.observeinc.com/en/latest/content/data-ingestion/forwarders/fluentbit.html). For now the functionality is pretty basic and it issues a POST request with the data records in [MessagePack](http://msgpack.org) (or JSON) format.  The following are the specfic HTTP parameters to employ
```

Comment:
Can you split sentences so they are on single lines? It makes it easier to comment on then.

You have double spacing and are missing a terminating full stop as well.

Reply:
Thanks, @patrick-stephens .  Made the change, how does this look?

# Observe

Observe employs the **http** output plugin, allowing you to flush your records [into Observe](https://docs.observeinc.com/en/latest/content/data-ingestion/forwarders/fluentbit.html).

For now the functionality is pretty basic and it issues a POST request with the data records in [MessagePack](http://msgpack.org) (or JSON) format.  

The following are the specfic HTTP parameters to employ:

## Configuration Parameters

Reply:
It's fine although you don't need to have an empty line between each: the main thing is that Github only lets you comment on a line-by-line basis so if a line has a whole paragraph on it then it can get confusing for review comments. Splitting sentences up means you can comment on a specific sentence rather than everything on the whole line.